src/routes/game/components/GameDialogs.vue

```diff
@@ -162,6 +165,9 @@ export default {
     resolveThree(cardId) {
       this.gameStore.requestResolveThree(cardId).then(this.clearSelection).catch(this.handleError);
     },
+    resolveFive(cardId) {
+      this.gameStore.requestResolveFive(cardId);
+    },
```

Comment:
I wonder if this is worth putting here or if we should just put it right inside the five dialog component

Reply:
Lets leave it here for now, for consistency